apps/judicial-system/backend/src/app/modules/court/court.service.ts

```diff
@@ -686,6 +687,38 @@ export class CourtService {
     }
   }
 
+  updateIndictmentCaseWithArraignmentDate(
+    user: User,
+    caseId: string,
+    courtName?: string,
+    courtCaseNumber?: string,
+    arraignmentDate?: Date,
+  ): Promise<unknown> {
+    try {
+      const subject = `${courtName} - ${courtCaseNumber} - þingfesting`
+      const content = JSON.stringify({ arraignmentDate })
+
+      return this.sendToRobot(
+        subject,
+        content,
+        RobotEmailType.INDICTMENT_CASE_ARRIGNMENT_DATE,
```

Comment:
I believe the type is only for our internal robot logs, but do we need to align on the name of the arraignment date property with Evolv? So it will be added to the desired target field in GoPro?